koko_controllers:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 500
  # Position Controllers ---------------------------------------
  joint_positions_controller:
    # TODO: resolve redundant constraint
    paired_constraints: [0, 1]
    endlink: 'roll_link'
    filter_length: 20
    type: koko_controllers/InverseDynamicsController
    root_name: koko
    zero_g_mode: false
    lift_joint:
      #p: 100
      p: 5.0
      d: 2.0
      id: 0.0
      max_torque: 30
      min_torque: -30
      max_angle: 8.5
      min_angle: -8.5
    roll_joint:
      #p: 100
      p: 5.0
      d: 2.0
      id: 0.0
      max_torque: 30
      min_torque: -30
      max_angle: 8.6
      min_angle: -8.6
